plugins/usbdmx/README.md

```diff
@@ -1,5 +1,37 @@
-USBDMX Plugin
```

Comment:
Can you split the dev info out into a readme developer file of some sort please, as we don't need it to appear in the UI for the users.

It probably makes sense to standardise plugins/osc/README in the same way at the same time.